hack/validate/golangci-lint.yml

```diff
@@ -24,6 +25,16 @@ linters:
       - docs
 
 linters-settings:
+  importas:
+    # Do not allow unaliased imports of aliased packages.
+    no-unaliased: true
+
+    alias:
+      # Enforce alias to prevent it accidentally being used instead of our
+      # own errdefs package (or vice-versa).
+      - pkg: github.com/containerd/containerd/errdefs
+        alias: cerrdefs
```

Comment:
FWIW, I picked `cerrdefs` as that looked to be the most common used, but `c8derrdefs`or `c8derrors` also works for me (I had to pick one 😂)